package/upgrade/upgrade_node.sh

```diff
@@ -586,6 +586,13 @@ command_single_node_upgrade() {
   # Add logging related kube-audit policy file
   patch_logging_event_audit
 
+  # During the RKE2 upgrade, the kube-api server will be restarted and fleet-agent will restart.
+  # The fleet agent might not clear applying charts state in this case.
+  # We stop the agent and let it gracefully handle the clearn up.
+  echo "Scale down the fleet agent..."
+  kubectl scale --replicas=0 deployment/fleet-agent -n cattle-fleet-local-system
```

Comment:
```
  name: fleet-agent-86dc84bdb7-pvw8r
  namespace: cattle-fleet-local-system


  serviceAccountName: fleet-agent
  terminationGracePeriodSeconds: 30
```

We can either scale it down, or simply wait all `bundles` to be ready. The `scale-down` may still have issue, given the grace period is only 30s.

will test it.